include/deal.II/matrix_free/matrix_free.h

```diff
@@ -534,7 +534,7 @@ class MatrixFree : public Subscriptor
    * pointers to the initialization function. Note that the finite element
    * underlying an DoFHandler must either be scalar or contain several copies
    * of the same element. Mixing several different elements into one
-   * @p FE_System is not allowed.
+   * FESystem is not allowed.
```

Comment:
I realize that this comment is outdated since #6300 or one of the related commits from #5667. Can you replace the part starting from `Note that the finite element underlying...` by the following text (I do not have the time right now to prepare it myself):
```
As an alternative, a system of several components can also be represented by a single
DoFHandler based on an FESystem. The prerequisite for this case is that each base element
of the FESystem must be compatible with the present class, such as the FE_Q or FE_DGQ
classes.
```

Reply:
Done.